src/components/About/About.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import styles from "./About.module.css";
+import classnames from 'classnames';
+import { Octokit } from "@octokit/rest";
+import Pagination from '@material-ui/lab/Pagination';
+
+const octokit = new Octokit();
+
+class About extends React.Component {
+  state = {
+    isLoading: true,
+    isError: false,
+    error: "",
+    repoList: [],
+    info: [],
+    currentPage: 1,
+    maxReposInPage: 2
+  }
+
+  componentDidMount() {
+    this.requestRepolist(this.state.maxReposInPage, this.state.currentPage);
+
+    octokit.users.getByUsername({
+      username: "tytytyw"
+        }).then(({ data }) => {
+          this.setState({
+            info: data,
+        });
+    })
+    .catch( (error) => {
+      this.setState({
+        bioIsLoading: false,
+        isError: true,
+        error: error.message
+      });
+    });
+  }
+
+  requestRepolist = (maxReposInPage, selectedPage) => {
+  octokit.repos.listForUser({
+    username: 'tytytyw',
+    per_page: maxReposInPage,
```

Comment:
Codacy found an issue: [Identifier 'per_page' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=6099781)